src/shims/env.rs

```diff
@@ -54,33 +63,109 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
     fn getenv(&mut self, name_op: OpTy<'tcx, Tag>) -> InterpResult<'tcx, Scalar<Tag>> {
         let this = self.eval_context_mut();
         let target_os = this.tcx.sess.target.target.target_os.as_str();
-        assert!(target_os == "linux" || target_os == "macos", "`{}` is only available for the UNIX target family");
+        assert!(target_os == "linux" || target_os == "macos", "`getenv` is only available for the UNIX target family");
```

Comment:
This doesn't appear to use `to_lowercase`.

Reply:
Yeah and it shouldn't.